src/containers/Articles/Article/Bookmark.jsx

```diff
@@ -0,0 +1,75 @@
+import React, {Component} from 'react';
+import {connect} from "react-redux";
+import PropTypes from "prop-types";
+import {withRouter} from "react-router-dom";
+import Bookmark from "../../../components/article/Bookmark";
+import {
+    bookmarkAction,
+    undoBookmarkAction
+} from "../../../store/actions/async/ArticleActions";
+import {isLoggedIn} from "../../../store/actions/async";
+
+export class BookmarkContainer extends Component{
+    bookmarkArticle() {
```

Comment:
@Karuhanga the team had agreed on using arrow functions conventions at the start of the project. is there a specific reason you choose to still use ES5. there more lines in the code that have this as well.

Reply:
Thanks for the review @missvicki. This is a 

> Nice work here @Karuhanga. I however tried hitting the bookmarks link on the profile drop-down and nothing happens. It just keeps on the current page.



> Could you please return response messages to the user to let them know if they've bookmarked the article or unbookmarked it. Otherwise, nice work

Thanks @missvicki. In my opinion, the visual indicator is sufficient for this. It would not be good `UX` to `toast` on `every` `action`.